packages/server/graphql/types/NewMeeting.ts

```diff
@@ -119,6 +119,10 @@ export const newMeetingFields = () => ({
     description: 'true if should show the org the conversion modal, else false',
     resolve: ({showConversionModal}: {showConversionModal: boolean}) => !!showConversionModal
   },
+  summary: {
```

Comment:
I decided to add `summary` here instead of `MeetingRetrospective` because we'll probably want to add this to other meeting types if the feature is successful